packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -1983,13 +2065,6 @@ const TESTS_WITH_PAYPAL_AS_PAYMENT_PROVIDER: [
   [
     'subscriptionSubsequentInvoiceDiscountEmail',
     new Map<string, Test | any>([
-      [
-        'subject',
```

Comment:
Just realized I need to add this back - I was actually removing some of these `subject` tests due to them failing and me thinking we needed to check for this in the HTML now, but I realized a commit I'd cherry-picked in to start had removed `subject` from `emailConfig` which is why these were all failing. I painstakenly moved them all back manually since I'd added more tests in the same commit lol, I'll fix this one before merge.

Reply:
Since you bring this up, I wonder if we should throw an error in the render if the subject is missing. Since the subject = getText.. was phased out, having a nice fail hard / fail fast check might be some extra safety just in case there's a gap in these tests somehow.

Reply:
Good idea. I've added a note to #11960